### DISCUSSION TITLE
Preview files in program browsers (Like photoshop)

### DISCUSSION THREAD

**Discussion by ebmh** [QL-Win/QuickLook#1137] [Q&A] [answered]
Hi, is there a way to preview files when I open my file browser through photoshop or other programs? At the moment nothing happens when I press spacebar in files browsers which are opened in other programs.

Thanks!

**Comment by xupefei** [comment, marked as answer]
Check https://github.com/QL-Win/QuickLook/wiki/Differences-Between-Distributions. You may have to use the MSI version.
